backend/cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2025,7 +2025,7 @@ msgstr "Schlüssel entfernen"
 #: templates/settings/mfa/delete.html:22
 msgid ""
 "Once you remove the key you will need to use one of the other available keys "
-"to log into your account. Please make sure that you have at least one extra "
```

Comment:
Oh, this is the automatically generated part - so no need to change anything here. Just fix it in the `backend/cms/templates/settings/mfa/delete.html` file and run the translate.sh-script, this will also fix this part.